lib/chef_node.rb

```diff
@@ -125,7 +125,22 @@ def reindex_and_wait(entry)
         raise "Did not find #{entry[:fqdn]} in Chef index after 180 seconds!"
       end
 
+      # If running on a machine with /etc/chef-server/admin.pem
+      # will use that user
       def ridley
+        # use admin account if available
```

Comment:
Can you list operations where we need the pivotal/ admin user for the chef-server?

We should avoid this as possible as it keeps tying us on to having to run our own chef-server.

Reply:
Yes, we need a "human" account for interactive operations. We do not use the bootstrap server's identity in these updates to run things like knife, etc. for uploading roles, environment and cookbooks.